### PR TITLE
Fix defenses

### PR DESCRIPTION
**Pull request by Xanditbb**
Makes it so that the defense modifiers are added to defenses

### CODE REVIEW

File: module/documents/actor.mjs [hunk 1]
```diff
@@ -329,8 +329,8 @@ export class LHTrpgActor extends Actor {
     if (accessories.length > 0) {
       if (accessories.length <= 3) {
         for (let [i] of Object.entries(accessories)) {
-          pDefBonus += accessories[i].system.pdef ?? 0;
```
Comment (Tenyryas) [inline module/documents/actor.mjs:332]: Why are these removed?

**Review by Tenyryas — CHANGES_REQUESTED** [review]
Thanks for the fix, I completely overlooked that, but it seems that you do the total in the loop that adds the accessories `pdef` and `mdef` to `pDefBonus` and `mDefBonus` while also removing the addition of said accessories stats. Is there any reason for that?

**Comment by Xanditbb** [comment]
Not really, I didn't notice that I accidentally removed the accessory stats, that was an accident. Why I put the math there, I didn't notice that the loop was for accessories and just thought it'd be best to consolidate the math in one spot and messed up the placement.

**Comment by Xanditbb** [comment]
Fixed my commit

**Review by Tenyryas — APPROVED** [review]
Everything seems a-ok now. Thanks for your contribution!
